earthpy/__init__.py

```diff
@@ -10,3 +10,11 @@
 epsg = json.loads(
     resource_string("earthpy", "example-data/epsg.json").decode("utf-8")
 )
+
+
+def load_epsg():
```

Comment:
@mbjoseph  this works well BUT we need to document 

epsg["code"] returns ... which isn't explicitly built into the function but kind of is. SHould we just do that here and add some example "vignette" code to the function with how to call it?

so

```
>>> epsg["4326"]
expected return here
```

Reply:
@joemcglinchy  i just chatted with @mbjoseph 
we could do this

https://stackoverflow.com/questions/20227051/how-to-document-a-module-constant-in-python/20230473#20230473 
and just try `""" """` below the call to epsg. this would mean

1. ignore my first comment about removing the code above and 
2. remove this function and then
3. document the epsg element below the call to define it above.

let me know if that is clear as mud!